src/MarketingPageAcceptanceTests.Steps/appsettings.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "hubUrl": "http://localhost:4444/wd/hub",
+  "pbUrl": "https://host.docker.internal",
```

Comment:
As per comment above, this would need to get changed to 
`"mpUrl": "https://host.docker.internal/marketing/supplier/solution",`